### PR TITLE
Distributed EJB timers post

### PR DESCRIPTION
**Pull request by pferraro**
Blog post introducing https://issues.redhat.com/browse/WFLY-7628 feature.

### CODE REVIEW

**Review by asoldano — COMMENTED** [review]
Nice, thanks!

**Comment by pferraro** [comment]
@bstansberry Can this get in soon?

**Review by rhusar — APPROVED** [review]
👍  Awesome, very concise.

File: _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+[source,xml]
+----
+<subsystem xmlns="urn:jboss:domain:ejb3:10.0">
+	<!-- ... -->
+	<timer-service default-persistent-timer-management="persistent" default-transient-timer-management="transient"/>
+	<!-- ... -->
+</subsystem>
+----
+
+The `default-persistent-timer-management` and `default-transient-timer-management` that define the server-wide default timer management used for persistent and transient (i.e. non-persistent) timers referencing resources from the `distributable-ejb` subsystem (new to WildFly 27).
```
Comment (bstansberry) [inline _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc:41]: I didn't understand this. Perhaps...

s/that define/attributes define/g

s/referencing/by referencing/g

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks @pferraro -- I looked at this to check rendering and validity of links and all that looked good. While I was at it I made a few minor suggestions.

File: _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc (hunk 1 above)
Comment (pferraro) [reply _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc:41]: Oy. You are right. That sentence makes no sense - will fix.

File: _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Alternatively, persistent timers can use a replicated cache, or even an invalidation cache with a shared cache store - as all three cache types support the requisite consistent hash semantics to deterministically distribute timers among existing cluster members.
+
+The "transient" cache configuration uses a local cache configured with passivation to a file, but purges its store on restart (to conform to the non-persistent nature of these timers).
+
+Note that passivation thresholds are not configured by the cache configuration directly, but rather are configured by the associated timer-management configuration above.
+Users should also take care to retain the default transaction and locking configuration, as these semantics are crucial for preventing duplicate or missed timeouts as the result of server shutdown or topology changes.
+
+== Demonstrating Distributed Jakarta Enterprise Bean Timers
+
+The https://github.com/wildfly/quickstart/tree/main/ejb-timer#using-timer-service-within-a-cluster[ejb-timer quickstart] is the easiest way to see this feature in action.
```
[the remark this answers is not in the document]
Comment (pferraro) [reply _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc:109]: will do

File: _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc (hunk 1 above)
Comment (pferraro) [reply _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc]: Fixed.

File: _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc [hunk 3]
```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: 
+date: 2022-10-28
```
[the remark this answers is not in the document]
Comment (pferraro) [reply _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc]: Fixed.  I've changed the date to tomorrow and will merge tomorrow following tagging of quickstarts.

File: _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+----
+<subsystem xmlns="urn:jboss:domain:distributable-ejb:1.0">
+	<!-- ... -->
+	<infinispan-timer-management name="persistent" cache-container="ejb" cache="persistent" max-active-timers="10000"/>
+	<infinispan-timer-management name="transient" cache-container="ejb" cache="transient" max-active-timers="10000"/>
+</subsystem>
+----
+
+This configures two providers for timer management: one for persistent timers, and one for transient (i.e. non-persistent) timers.
+Here, users can configure the maximum number of timers created by a given EJB that should retain in memory at a given time, as well as the marshaller implementation (JBoss Marshalling, by default) used to serialize the "info" associated with a given Timer.
```
[the remark this answers is not in the document]
Comment (pferraro) [reply _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc]: Fixed.

File: _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (pferraro) [reply _posts/2022-10-28-DistributedJakartaEnterpriseBeanTimers.adoc]: Fixed.

**Pull request merged**